### ISSUE TITLE
Does not load in Glyphs 3

### ISSUE THREAD

**Issue by mekkablue** [mekkablue/ShowCoordinatesOfSelectedNodes#8]
Build 3080, Console.app:

```
19:54:08.141911+0200	Plugin loading: Problem with plugin at path (Info): /Users/mekka/Library/Application Support/Glyphs 3/Repositories/ShowCoordinatesOfSelectedNodes/ShowCoordinatesOfSelectedNodes.glyphsReporter
```

**Comment by mekkablue** [comment]
@schriftgestalt cannot figure this one out. I cannot make the plug-in load. I believe I completely synced everything with the reporter template in the GlyphsSDK. Must be something obvious I am missing.

**Comment by schriftgestalt** [comment]
fixed in 80a64efdd76c698a38eaa559a90d272c2be6a687
